### PR TITLE
build: change extracted image for macOS

### PR DESCRIPTION
**Pull request by gdams**
THIS IS A BREAKING CHANGE

This will change the directory layout of the binaries to be the apple layout which is built at the same time as the main `JDK` and `JRE` images

### CODE REVIEW

File: configureBuild.sh [hunk 1]
```diff
@@ -127,10 +127,22 @@ setVariablesForConfigure() {
     [ "$openjdk_core_version" == "${JDKHEAD_CORE_VERSION}" ]; then
     local jdk_path="jdk"
     local jre_path="jre"
+    case "${BUILD_CONFIG[OS_KERNEL_NAME]}" in
```
Comment (karianna) [inline configureBuild.sh]: Is that correct for Java 9+?

File: configureBuild.sh [hunk 2]
```diff
@@ -127,10 +127,22 @@ setVariablesForConfigure() {
...
+    "darwin")
+      local jdk_path="jdk-bundle/jdk-*.jdk"
```
Comment (karianna) [inline configureBuild.sh]: I assume `local` in bash is local to the scope it's in (i.e. the if statement)

File: configureBuild.sh (hunk 1 above)
Comment (gdams) [reply configureBuild.sh]: yup

File: configureBuild.sh (hunk 2 above)
Comment (gdams) [reply configureBuild.sh]: yup

**Pull request merged**